### PR TITLE
Explained a bit more what SwipeNavigate does

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

**Comment by kristianhasselknippe** [comment]
This is just adding more small talk. Please explain about the properties of `SwipeNavigate` instead.

File: Fuse/09 - Navigation.md [hunk 1]
```diff
@@ -430,8 +430,11 @@ TODO -->
 
 ## $(SwipeNavigate)
 
-`SwipeNavigate` is used to connect swiping gestures to @(Navigation).
-`SwipeNavigate` has three properties which modify its behavior: @(SwipeDirection), @(SwipeEnds) and @(VelocityThreshold).
+`SwipeNavigate` is used to connect swiping gestures to @(Navigation). `SwipeNavigate` has three properties which modify its behavior:
+
+* @(SwipeDirection) - Specifies the direction the user has to swipe to navigate "forwards".
```
Comment (sebbert) [inline Fuse/09 - Navigation.md]: How about...
Specifies the direction the user has to swipe **in order** to navigate "forward".

File: Fuse/09 - Navigation.md [hunk 2]
```diff
@@ -430,8 +430,11 @@ TODO -->
 
 ## $(SwipeNavigate)
 
-`SwipeNavigate` is used to connect swiping gestures to @(Navigation).
-`SwipeNavigate` has three properties which modify its behavior: @(SwipeDirection), @(SwipeEnds) and @(VelocityThreshold).
+`SwipeNavigate` is used to connect swiping gestures to @(Navigation). `SwipeNavigate` has three properties which modify its behavior:
```
Comment (sebbert) [inline Fuse/09 - Navigation.md]: I'd suggest adding a sentence or so about the most common use case (i.e. horizontally stacked pages)

File: Fuse/09 - Navigation.md [hunk 3]
```diff
@@ -430,8 +430,11 @@ TODO -->
 
 ## $(SwipeNavigate)
 
-`SwipeNavigate` is used to connect swiping gestures to @(Navigation).
-`SwipeNavigate` has three properties which modify its behavior: @(SwipeDirection), @(SwipeEnds) and @(VelocityThreshold).
+`SwipeNavigate` is used to connect swiping gestures to @(Navigation). This is used internally by @(EdgeNavigator), but you would be interrested in it e.g. if you want horizontally stacked pages. `SwipeNavigate` has three properties which modify its behavior:
```
Comment (sebbert) [inline Fuse/09 - Navigation.md]: interested, not interrested

File: Fuse/09 - Navigation.md [hunk 4]
```diff
@@ -430,8 +430,11 @@ TODO -->
...
+
+* @(SwipeDirection) - Specifies the direction the user has to swipe in order to navigate "forwards".
+* @(SwipeEnds) - Decide how the animation handles there not being a next or previous element to navigate to. `Closed` hard-limits the animation to the end of the last element, `Open` let's the user swipe unhindered, `Short` creates resistance to the animation, effectively slowing the animation down the further the user swipes.
```
Comment (sebbert) [inline Fuse/09 - Navigation.md]: Decide -> Decides/Specifies/Controls or similar

**Pull request merged**